release/pom.xml

```
@@ -1,10 +1,9 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>it.geosolutions.mapstore</groupId>
   <artifactId>mapstore-bin-release</artifactId>
   <packaging>pom</packaging>
-  <version>${mapstore2.version}</version>
+  <version>1.4-SNAPSHOT</version>
```

I think the release need to keep this version. The binary should generate the proper zip package with the version passed.

```suggestion
  <version>${mapstore2.version}</version>
```